sdk/monitor/monitor-query/src/metricsQueryClient.ts

```diff
@@ -34,15 +34,21 @@ import {
   convertResponseForMetrics,
   convertResponseForMetricsDefinitions,
 } from "./internal/modelConverters";
-import { SDK_VERSION } from "./constants";
-const defaultMetricsScope = "https://management.azure.com/.default";
```

Comment:
Hi,
Previous value of defaultMetricsScope ("https://management.azure.com/.default") is different than KnownMonitorAudience.AzurePublicCloud ("https://monitor.azure.com"). Is this expected ?

Reply:
Yes. This is expected and made to be consistent with other SDKs